src/tests/diff-details/in/bar/transformation/keboola.python-transformation-v2/python-transformation/blocks/001-block/001-code-x/code.py

```diff
@@ -0,0 +1 @@
+print(100)
```

Comment:
V tomto PR je vela zmien v testoch.
Je to preto, lebo nova feature, editacia transformacii ako nativnych suborov sa tyka takmer vsetkych prikazov.
V integracnych testoch boli uz dopredu pripravne cases s transformaciami, ... teda, teraz staci subory uz iba editovat.

Vznili nove subory `code.*`, `meta.json` ... a z `config.json` zmizol kluc `parameters.blocks`, ... ktory bol nahradeny subormi na disku.